src/applications/appeals/996/tests/hlr-keyboard-only.cypress.spec.js

```diff
@@ -110,8 +110,10 @@ describe('Higher-Level Review keyboard only navigation', () => {
         'include',
         chapters.informalConference.pages.requestConference.path,
       );
+      // eslint-disable-next-line cypress/no-unnecessary-waiting
```

Comment:
Added a `cy.wait(250)` to allow the H3 to be focused _before_ using the `tabToElement` command. Without it, we get a `cy.get(':focus')` not found error

Reply:
I'm not a huge fan of hardcoded waits, but at least it is a short one. I think this may be related to the keyboard navigation extensions being used here.